andino_firmware/src/app.cpp

```diff
@@ -199,114 +195,142 @@ void App::reset_command() {
       break;
-    case PIN_MODE:
-      if (arg2 == 0)
-        pinMode(arg1, INPUT);
-      else if (arg2 == 1)
-        pinMode(arg1, OUTPUT);
-      Serial.println("OK");
+    case Commands::ksetMotorsSpeed:
+      cmd_set_motors_speed(argv1, argv2);
       break;
-    case READ_ENCODERS:
-      Serial.print(left_encoder_.read());
-      Serial.print(" ");
-      Serial.println(right_encoder_.read());
+    case Commands::ksetMotorsPwm:
+      cmd_set_motors_pwm(argv1, argv2);
       break;
-    case RESET_ENCODERS:
-      left_encoder_.reset();
-      right_encoder_.reset();
-      left_pid_controller_.reset(left_encoder_.read());
-      right_pid_controller_.reset(right_encoder_.read());
-      Serial.println("OK");
-      break;
-    case MOTOR_SPEEDS:
-      /* Reset the auto stop timer */
-      lastMotorCommand = millis();
-      if (arg1 == 0 && arg2 == 0) {
-        left_motor_.set_speed(0);
-        right_motor_.set_speed(0);
-        left_pid_controller_.reset(left_encoder_.read());
-        right_pid_controller_.reset(right_encoder_.read());
-        left_pid_controller_.enable(false);
-        right_pid_controller_.enable(false);
-      } else {
-        left_pid_controller_.enable(true);
-        right_pid_controller_.enable(true);
-      }
-      // The target speeds are in ticks per second, so we need to convert them
-      // to ticks per PID_INTERVAL
-      left_pid_controller_.set_setpoint(arg1 / Constants::kPidRate);
-      right_pid_controller_.set_setpoint(arg2 / Constants::kPidRate);
-      Serial.println("OK");
-      break;
-    case MOTOR_RAW_PWM:
-      /* Reset the auto stop timer */
-      lastMotorCommand = millis();
-      left_pid_controller_.reset(left_encoder_.read());
-      right_pid_controller_.reset(right_encoder_.read());
-      // Sneaky way to temporarily disable the PID
-      left_pid_controller_.enable(false);
-      right_pid_controller_.enable(false);
-      left_motor_.set_speed(arg1);
-      right_motor_.set_speed(arg2);
-      Serial.println("OK");
-      break;
-    case UPDATE_PID:
-      /* Example: "u 30:20:10:50" */
-      while ((str = strtok_r(p, ":", &p)) != NULL) {
-        pid_args[i] = atoi(str);
-        i++;
-      }
-      left_pid_controller_.set_tunings(pid_args[0], pid_args[1], pid_args[2], pid_args[3]);
-      right_pid_controller_.set_tunings(pid_args[0], pid_args[1], pid_args[2], pid_args[3]);
-      Serial.print("PID Updated: ");
-      Serial.print(pid_args[0]);
-      Serial.print(" ");
-      Serial.print(pid_args[1]);
-      Serial.print(" ");
-      Serial.print(pid_args[2]);
-      Serial.print(" ");
-      Serial.println(pid_args[3]);
-      Serial.println("OK");
+    case Commands::ksetPidsTuningGains:
+      cmd_set_pid_tuning_gains(argv1, argv2);
       break;
     default:
-      Serial.println("Invalid Command");
+      cmd_unknown(argv1, argv2);
       break;
   }
 }
 
+void App::cmd_unknown(const char* arg1, const char* arg2) {
+  (void)arg1;
+  (void)arg2;
```

Comment:
nit: 
1. static_cast can be used here to be more `cpp`
2. I think that simply omitting the parameters hints the compiler that the args won't be used.
```cpp
void App::cmd_unknown(const char* , const char* ) {
```

Reply:
Done.